scripts/tests/twister_blackbox/test_data/boards/dummy_arch/dummy_platform/board.yml

```diff
@@ -0,0 +1,4 @@
+board:
+  name: dummy
+  vendor: others
+  socs: []
```

Comment:
You cannot use `dummy_vendor`; Zephyr has a vendor whitelist.

Reply:
Also, why would we want to fill out the `socs` list? It is not necessary for the test and requires adding new files, to avoid the `ERROR: SoC 'dummy_soc' is not found, please ensure that the SoC exists and that soc-root containing 'dummy_soc' has been correctly defined.`

Reply:
Those new files would also need to be outside the `TEST_DATA` folder, as `soc-root`s are hard-coded inside Twister and cannot be changed via a flag.

Reply:
I wonder should this twister blackbox test use `subsys/testsuite` folder then the same way as unit_testing does (see also #69786)

Reply:
That PR still uses the legacy HWMv1, doesn't it?

Reply:
OK, I have found `unit_testing` SoCs in `$ZEPHYR_BASE/scripts/pylib/twister/soc`. That path is one of the two hardcoded paths and I'll be able to put the `dummy_soc` definitions there.
However, the PR you've linked changes the hard-coded path from `$ZEPHYR_BASE/scripts/pylib/twister/soc` to `$ZEPHYR_BASE/subsys/testsuite` (still hardcoded, however). This means that these two PRs would be conflicting.

Do we want to use the current path (`$ZEPHYR_BASE/scripts/pylib/twister/soc`) and notify the other PR in case this one is merged first or use the proposed new path (`$ZEPHYR_BASE/subsys/testsuite/soc`) and wait for them to be merged first?

Or maybe even add another path to the hard-coded ones (`$TEST_DATA/soc` or something similar) so we can stay within `TEST_DATA` with our board and SoC definitions.

Reply:
It would be best to afford `soc_root` and `arch_root` the same treatment as `board_roots`; that is: option of adding more via Twister flags.